device/iot-device-samples/module-invoke-method-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ModuleInvokeMethodSample.java

```diff
@@ -82,21 +82,23 @@ else if (protocolStr.equalsIgnoreCase("mqtt_ws"))
         ModuleClient client = ModuleClient.createFromEnvironment(new UnixDomainSocketSample.UnixDomainSocketChannelImpl(), protocol);
         client.open(false);
 
-        MethodRequest methodRequest = new MethodRequest(methodName, methodPayload);
-        MethodResult result;
+        // Along with String as the type of "methodPayload" here, the type can also be Null/Primitive type/Array/List/Map/custom type.
+        // Please also refer to https://github.com/Azure/azure-iot-sdk-java/blob/main/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/edge/DirectMethodRequest.java
```

Comment:
Link in this comment will be available only after the changes in this PR get merged into main.